ASP.NET MVC/Perform_OCR_MVC/packages.config

```diff
@@ -1,20 +1,23 @@
 <packages>
-  <package id="Antlr" version="3.5.0.2" targetFramework="net472" />
-  <package id="bootstrap" version="5.0.0" targetFramework="net472" />
-  <package id="jQuery" version="3.7.1" targetFramework="net472" />
-  <package id="jQuery.Validation" version="1.21.0" targetFramework="net472" />
-  <package id="Microsoft.AspNet.Mvc" version="5.2.7" targetFramework="net472" />
-  <package id="Microsoft.AspNet.Razor" version="3.2.7" targetFramework="net472" />
-  <package id="Microsoft.AspNet.Web.Optimization" version="1.1.3" targetFramework="net472" />
-  <package id="Microsoft.AspNet.WebPages" version="3.2.7" targetFramework="net472" />
-  <package id="Microsoft.CodeDom.Providers.DotNetCompilerPlatform" version="2.0.1" targetFramework="net472" />
-  <package id="Microsoft.jQuery.Unobtrusive.Validation" version="4.0.0" targetFramework="net472" />
-  <package id="Microsoft.Web.Infrastructure" version="1.0.0.0" targetFramework="net472" />
-  <package id="Modernizr" version="2.8.3" targetFramework="net472" />
-  <package id="Newtonsoft.Json" version="13.0.3" targetFramework="net472" />
-  <package id="Syncfusion.Compression.Base" version="20.4.0.38" targetFramework="net472" />
-  <package id="Syncfusion.Pdf.AspNet.Mvc5" version="26.2.8" targetFramework="net472" />
-  <package id="Syncfusion.Pdf.OCR.AspNet.Mvc5" version="26.2.8" targetFramework="net472" />
-  <package id="WebGrease" version="1.6.0" targetFramework="net472" />
+  <package id="Antlr" version="3.5.0.2" targetFramework="net462" />
+  <package id="bootstrap" version="5.0.0" targetFramework="net462" />
+  <package id="jQuery" version="3.7.1" targetFramework="net462" />
+  <package id="jQuery.Validation" version="1.21.0" targetFramework="net462" />
+  <package id="Microsoft.AspNet.Mvc" version="5.2.7" targetFramework="net462" />
+  <package id="Microsoft.AspNet.Razor" version="3.2.7" targetFramework="net462" />
+  <package id="Microsoft.AspNet.Web.Optimization" version="1.1.3" targetFramework="net462" />
+  <package id="Microsoft.AspNet.WebPages" version="3.2.7" targetFramework="net462" />
+  <package id="Microsoft.CodeDom.Providers.DotNetCompilerPlatform" version="2.0.1" targetFramework="net462" />
+  <package id="Microsoft.jQuery.Unobtrusive.Validation" version="4.0.0" targetFramework="net462" />
+  <package id="Microsoft.Web.Infrastructure" version="1.0.0.0" targetFramework="net462" />
+  <package id="Modernizr" version="2.8.3" targetFramework="net462" />
+  <package id="Newtonsoft.Json" version="13.0.3" targetFramework="net462" />
+  <package id="Syncfusion.Compression.Base" version="27.1.53" targetFramework="net462" />
+  <package id="Syncfusion.Licensing" version="27.1.53" targetFramework="net462" />
+  <package id="Syncfusion.Pdf.AspNet.Mvc5" version="27.1.53" targetFramework="net462" />
+  <package id="Syncfusion.Pdf.OCR.AspNet.Mvc5" version="27.1.53" targetFramework="net462" />
+  <package id="Syncfusion.Pdf.OCR.Wpf" version="27.1.53" targetFramework="net462" />
```

Comment:
May I know the reason for using this WPF package?

Reply:
removed